string-manipulation-1/src/is-vowel.js

```diff
@@ -5,7 +5,11 @@
 // function isVowel(char) { }
 // include the string 'aeiouAEIOU' to include all of the vowels that are lower and uppercased.
 // .includes() function will return true or false if char matches any of the strings so then write: // return 'aeiouAEIOU'.includes(char)
+// ^^ old method
 
+// Set.prototype.has() method returns a boolean to indicate whether an element with the specified value exists in the Set object or not.
+// In this case, we will set Set() to 'AEIOU' and 'aeiou' then assign it a new variable for it to be returned.
 function isVowel(char) {
-  return 'aeiouAEIOU'.includes(char);
+  var vowels = new Set('AaEeIiOoUu');
```

Comment:
I didn't teach you about `Set`s. If you can tell me exactly how it works, you can keep this, but otherwise, you should make this logic yourself.